src/ape_plugins/_cli.py

```diff
@@ -156,6 +156,7 @@ def add(cli_ctx, plugin, version, skip_confirmation):
 @ape_cli_context()
 @skip_confirmation_option("Don't ask for confirmation to install the plugins")
 def install(cli_ctx, skip_confirmation):
+    is_insall_fail = False
```

Comment:
Can we rename this to something like `install_failures_occurred` or `failures_occurred`.

It should kind-of read like english